pkg/env/env.go

```diff
@@ -160,70 +172,133 @@ func (e *testEnv) AfterEachTest(funcs ...types.TestEnvFunc) types.Environment {
 	return e
 }
 
-// Test executes a feature test from within a TestXXX function.
-//
-// Feature setups and teardowns are executed at the same *testing.T
-// contextual level as the "test" that invoked this method. Assessments
-// are executed as a subtests of the feature.  This approach allows
-// features/assessments to be filtered using go test -run flag.
-//
-// Feature tests will have access to and able to update the context
-// passed to it.
-//
-// BeforeTest and AfterTest operations are executed before and after
-// the feature is tested respectively.
-func (e *testEnv) Test(t *testing.T, testFeatures ...types.Feature) {
+// panicOnMissingContext is used to check if the test Env has the right context setup
```

Comment:
Nit: it checks not for the `right` context, but that there is a `non-nil` context.